crates/dslab-mp/tests/pending_events.rs

```diff
@@ -9,12 +9,12 @@ fn test_mc_time() {
     let b = McTime::from(0.0);
     assert!(b <= a);
     assert!(a <= b);
```

Comment:
а тут тогда наверное так может https://docs.rs/more-asserts/latest/more_asserts/macro.assert_le.html?
```suggestion
    assert_le!(a, b);
```

Reply:
В данном случае мне удобнее сразу видеть неравенство